README.md

```diff
@@ -567,7 +567,10 @@ It's recommended that you set up your codemod project to all debugging via the V
             ],
             "console": "internalConsole",
             "sourceMaps": true,
-            "outFiles": []
+            "outFiles": [],
+            "windows": {
+                "program": "${workspaceRoot}/node_modules/jscodeshift/bin/jscodeshift.js",
```

Comment:
Try `${workspaceRoot}/node_modules/.bin/jscodeshift.cmd` and see if that works - for more consistency with how it's done for other platforms.

Reply:
Thank you for your reply.
When I tried to launch debugger using it, I got the following error in DEBUG CONSOLE:

```
C:\Program Files\Volta\node.exe --nolazy .\node_modules\.bin\jscodeshift.cmd --dry --print -t transform.js --parser babel --run-in-band C:\projects\jscodeshift\t.js
Uncaught SyntaxError C:\projects\jscodeshift\node_modules\.bin\jscodeshift.cmd:1
@IF EXIST "%~dp0\node.exe" (
^

SyntaxError: Invalid or unexpected token
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1178:20)
    at <anonymous> (node:internal/modules/cjs/loader:1220:27)
    at <anonymous> (node:internal/modules/cjs/loader:1310:10)
    at <anonymous> (node:internal/modules/cjs/loader:1119:32)
    at <anonymous> (node:internal/modules/cjs/loader:960:12)
    at executeUserEntryPoint (node:internal/modules/run_main:81:12)
    at <anonymous> (node:internal/main/run_main_module:23:47)
Process exited with code 1
```

It seems that vscode runs `.\node_modules\.bin\jscodeshift.cmd` using node.

When I ran `.\node_modules\.bin\jscodeshift.cmd --dry --print -t transform.js --parser babel --run-in-band C:\projects\jscodeshift\t.js` it was successfull but I couldn't find any settings in launch.json to do this.